lib/src/features/user/ui/profile_screen.dart

```diff
@@ -231,47 +234,65 @@ class PerfCards extends StatelessWidget {
         itemBuilder: (context, index) {
           final perf = userPerfs[index];
           final userPerf = account.perfs[perf]!;
+          final bool isPerfWithoutStats =
+              [Perf.puzzle, Perf.storm].contains(perf);
           return SizedBox(
             height: 100,
             width: 100,
-            child: PlatformCard(
-              child: Padding(
-                padding: const EdgeInsets.all(8.0),
-                child: Column(
-                  mainAxisAlignment: MainAxisAlignment.spaceAround,
-                  children: [
-                    Text(
-                      perf.shortName,
-                      style: TextStyle(color: textShade(context, 0.7)),
-                    ),
-                    Icon(perf.icon, color: textShade(context, 0.6)),
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.center,
-                      children: [
-                        Text(
-                            '${userPerf.rating}${userPerf.provisional == true || userPerf.ratingDeviation > kProvisionalDeviation ? '?' : ''}',
-                            style: kBold),
-                        const SizedBox(width: 3),
-                        if (userPerf.progression != 0) ...[
-                          Icon(
-                            userPerf.progression > 0
-                                ? LichessIcons.arrow_full_upperright
-                                : LichessIcons.arrow_full_lowerright,
-                            color: userPerf.progression > 0
-                                ? LichessColors.good
-                                : LichessColors.red,
-                            size: 12,
-                          ),
-                          Text(userPerf.progression.abs().toString(),
-                              style: TextStyle(
-                                  color: userPerf.progression > 0
-                                      ? LichessColors.good
-                                      : LichessColors.red,
-                                  fontSize: 12)),
+            child: InkWell(
+              splashFactory: isPerfWithoutStats
+                  ? NoSplash.splashFactory
+                  : InkSplash.splashFactory,
+              customBorder: kPlatformCardBorder,
+              onTap: isPerfWithoutStats
+                  ? () {}
```

Comment:
It is better to put `null` instead of `() {}`. In the case of a button this is the way flutter will mark it as `disabled`. Not sure it applies to `InkWell` though.